dubbo-spring-boot/dubbo-spring-boot-actuator/src/main/java/org/apache/dubbo/spring/boot/actuate/autoconfigure/DubboEndpointAnnotationAutoConfiguration.java

```diff
@@ -45,41 +46,47 @@
 
     @Bean
     @ConditionalOnMissingBean
+    @ConditionalOnAvailableEndpoint
```

Comment:
`@ConditionalOnAvailableEndpoint` is available sine Spring Boot 2.2.0, see https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/autoconfigure/endpoint/condition/ConditionalOnAvailableEndpoint.html
If we reference `@ConditionalOnAvailableEndpoint` directly, it may cause loading failed on Spring Boot version < 2.2.0.

Reply:
It seems missing annotation class will not cause ClassNotFoundException, please check it on Spring Boot version < 2.2.0.